mhs/common/mhs_common/messages/ebxml_request_envelope.py

```diff
@@ -131,10 +134,9 @@ def from_string(cls, headers: Dict[str, str], message: str) -> EbxmlRequestEnvel
         cls._extract_more_values_from_xml_tree(xml_tree, extracted_values)
 
         cls._add_descriptions_to_attachments(xml_tree, attachments)
+        extracted_values[EBXML] = ebxml_part
         extracted_values[ATTACHMENTS] = attachments
 
-        logger.info('Extracted {extracted_values} from message', fparams={'extracted_values': extracted_values})
```

Comment:
This would cause medical data to be dropped as info logs.
Also this is not needed even in debug logs as it's basically the whole message transformed into a dictionary